apps/api/src/app/environments-v1/e2e/environments.controller.e2e.ts

```
@@ -24,6 +25,28 @@ describe('Env Controller', async () => {
 
+    it('should default the in-app integration HMAC to ON for non-dev environments (NV-7593)', async () => {
+      await session.updateOrganizationServiceLevel(ApiServiceLevelEnum.BUSINESS);
+
+      const { environmentRequestDto } = generateRandomEnvRequest();
+      const createdEnv = await novuClient.environments.create(environmentRequestDto);
+      expect(createdEnv.result).to.be.ok;
+
+      const integrationRepository = new IntegrationRepository();
+      const inAppIntegration = await integrationRepository.findOne({
+        _organizationId: session.organization._id,
+        _environmentId: createdEnv.result?.id!,
```

Biome (2.4.15)

[error] Forbidden non-null assertion after optional chaining.

(lint/suspicious/noNonNullAssertedOptionalChain)